src/Evolution/Protocols.hpp

```diff
@@ -0,0 +1,33 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Utilities/TypeTraits/CreateHasTypeAlias.hpp"
+
+namespace evolution {
+/// \ref protocols related to evolution systems
+namespace protocols {
+
+namespace detail {
+CREATE_HAS_TYPE_ALIAS(import_fields)
+}  // namespace detail
+
+/*!
+ * \ingroup ProtocolsGroup
+ * \brief Indicates the `ConformingType` provides compile-time information for
```

Comment:
My intention is to not couple the numeric data importing code to the system, but allow specifying the fields to be imported by any class that exposes the `import_fields`. So instead of using the `evolution::NumericInitialData<System>` class you could provide a class in the Metavariables that, for example, adds a bunch of diagnostic fields that should be imported from the volume data file to the list of system fields. If the DgElementArray will only ever need the list of fields to import then I could just pass that, but I'm thinking we may want to adjust the input file group or the callback action (e.g. for data verification) in the Metavariables. If we want to do that we can change the protocol and the classes that conform to it instead of changing template parameters in a bunch of places. Does that make sense?